biojava-aa-prop/src/main/java/org/biojava/nbio/aaproperties/PeptideProperties.java

```diff
@@ -564,7 +564,7 @@ public static final double getEnrichment(String sequence, String aminoAcidCode){
 	 * 		a protein sequence consisting of non-ambiguous characters only
 	 * @return the array of charges of amino acids in the protein (1 if amino acid is positively charged, -1 if negatively charged, 0 if not charged)
 	 */
-	public static final int[] getChargesOfAminoAcidsInProtein(String sequence) {
+	public static final int[] getChargesOfAminoAcids(String sequence) {
```

Comment:
It will not be consistent with other methods of the class that work with sequence input as String